src/libhirte/common/cfg.c

```diff
@@ -338,3 +339,95 @@ const char *cfg_dump(struct config *config) {
         }
         return cfg_info;
 }
+
+int cfg_agent_def_conf(struct config *config) {
+        int result = 0;
+
+        result = cfg_set_default_section(config, CFG_SECT_AGENT);
+        if (result != 0) {
+                fprintf(stderr,
+                        "Error setting default section for agent '%s', error code '%s'.\n",
+                        CFG_SECT_AGENT,
+                        strerror(-result));
+                return false;
+        }
+
+        // NodeName
+        if (cfg_set_value(config, CFG_NODE_NAME, get_hostname()) != 0) {
+                return -1;
+        }
+
+        // ManagerHost
+        if (cfg_set_value(config, CFG_MANAGER_HOST, HIRTE_DEFAULT_HOST) != 0) {
+                return -1;
+        }
+
+        // HeartbeatInterval
+        if (cfg_set_value(config, CFG_HEARTBEAT_INTERVAL, AGENT_HEARTBEAT_INTERVAL_MSEC) != 0) {
+                return -1;
+        }
+
+        // ManagerPort
+        if (cfg_set_value(config, CFG_MANAGER_PORT, HIRTE_DEFAULT_PORT) != 0) {
+                return -1;
+        }
+
+        // Loglevel
+        const char *LOG_LEVEL_INFO = "INFO";
+        if (cfg_set_value(config, CFG_LOG_LEVEL, LOG_LEVEL_INFO) != 0) {
+                return -1;
+        }
+
+        // LogTarget
+        const char *LOG_TARGET_STDERR = "stderr";
+        if (cfg_set_value(config, CFG_LOG_TARGET, LOG_TARGET_STDERR) != 0) {
+                return -1;
+        }
+
+        // LogIsQuiet
+        if (cfg_set_value(config, CFG_LOG_IS_QUIET, NULL) != 0) {
+                return -1;
+        }
+        return 0;
+}
+
+int cfg_manager_def_conf(struct config *config) {
+        int result = 0;
+
+        result = cfg_set_default_section(config, CFG_SECT_HIRTE);
+        if (result != 0) {
+                fprintf(stderr,
+                        "Error setting default section for agent '%s', error code '%s'.\n",
+                        CFG_SECT_HIRTE,
+                        strerror(-result));
+                return false;
```

Comment:
```suggestion
                return result;
```
The caller is already doing a `fprintf`.